### PR TITLE
A few small metadata updates and new images

### PR DESCRIPTION
**Pull request by mhoogcarspel**

### CODE REVIEW

**Review by Cebion — DISMISSED** [review]

**Review by Slayer366 — APPROVED** [review]

File: ports/arkovstower/port.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
       "Troidem"
     ],
     "desc": "This roguelike puts you in the shoes of an unnamed adventurer who just woke up on the top level of Arkov's Tower. Try to fight your way down and escape it - or die trying. Move your player trough numerous procedural generated levels. Fight numerous enemies including some unique bosses. The turn based combat system allows you to take your time and plan every move carefully. Upgrade your character with all sorts of items found in chests or dropped by enemies. This game is highly customizable for people with pixel art and/or programming experience.",
-    "inst": "Download the game https://store.steampowered.com/app/2688600/Descent_from_Arkovs_tower_Prologue/ and copy the exe file into game folder.",
+    "inst": "Download [Descent from Arkov's tower Prologue](https://store.steampowered.com/app/2688600/Descent_from_Arkovs_tower_Prologue/) from Steam and copy the exe file into game folder.",
```
Comment (kloptops) [inline ports/arkovstower/port.json:15]: These urls wont work properly in the PortMaster-GUI. :(

**Review by kloptops — CHANGES_REQUESTED** [review]
Just one small change. 😄

File: ports/arkovstower/port.json (hunk 1 above)
Comment (mhoogcarspel) [reply ports/arkovstower/port.json:15]: and the way it is, it shows up like this on the website :/ 
image
So, I'll take it out and we need to look at the markdown to html for the website?

Comment (christopher-roelofs) [reply ports/arkovstower/port.json:15]: > These urls wont work properly in the PortMaster-GUI. :(

We could add mkd_inst and mkd_desc and have the website use them if they are set.

**Comment by kloptops** [comment]
@mhoogcarspel we can now use two new tags `desc_md` and `inst_md` tags, they support markdown for the website, so you can change it to the following:

```json
   "inst": "Download the game https://store.steampowered.com/app/2688600/Descent_from_Arkovs_tower_Prologue/ and copy the exe file into game folder.",
    "inst_md": "Download [Descent from Arkov's tower Prologue](https://store.steampowered.com/app/2688600/Descent_from_Arkovs_tower_Prologue/) from Steam and copy the exe file into game folder.",
```

This way we have the best of both worlds. :)

**Comment by mhoogcarspel** [comment]
> @mhoogcarspel we can now use two new tags `desc_md` and `inst_md` tags, they support markdown for the website, so you can change it to the following:

very fancy!

**Review by kloptops — APPROVED** [review]

**Pull request merged**